kubernetes/base/chroma/deployment.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: chroma-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: chroma
+  template:
+    metadata:
+      labels:
+        app: chroma
+    spec:
+      containers:
+      - name: kg-llm-chroma
+        image: ghcr.io/chroma-core/chroma:0.3.23
+        command:
+        - uvicorn
+        args: ["chromadb.app:app", "--reload", "--workers", "1", "--host", "0.0.0.0", "--port", $(CHROMA_PORT), "--log-config", "log_config.yml"]
+        env:
+        - name: CHROMA_DB_IMPL
+          value: "clickhouse"
+        - name: CLICKHOUSE_HOST
+          value: "localhost"
+        - name: CLICKHOUSE_PORT
+          value: clickhouse_http_port_placeholder
+        - name: CHROMA_PORT
+          value: chroma_port_placeholder
+        ports:
+        - containerPort: 0
+          name: chroma-port
+        volumeMounts:
+        - name: index-data
+          mountPath: /index_data
+      - name: kg-llm-clickhouse
+        image: clickhouse/clickhouse-server:22.9-alpine
+        env:
+        - name: ALLOW_EMPTY_PASSWORD
+          value: "yes"
+        - name: CLICKHOUSE_TCP_PORT
+          value: $(CLICKHOUSE_TCP_PORT)
+        - name: CLICKHOUSE_HTTP_PORT
+          value: $(CLICKHOUSE_HTTP_PORT)
+        volumeMounts:
+        - name: clickhouse-data
+          mountPath: /var/lib/clickhouse
+        - name: backups
+          mountPath: /backups
+        - name: config
+          mountPath: /etc/clickhouse-server/config.d/backup_disk.xml
```

Comment:
Chroma fails to insert data because clickhouse is not configured properly. IIUC, the clickhouse config files should be added into the container from the repo.

That's what they seem to do in the chroma docker-compose: https://github.com/chroma-core/chroma/blob/666bfc40f3e380d91df2fb0d99d6124c9e53d36e/docker-compose.yml#L41

These clickhouse config files seem to be needed:
* chroma users: https://github.com/chroma-core/chroma/blob/main/config/chroma_users.xml
* backup: https://github.com/chroma-core/chroma/blob/666bfc40f3e380d91df2fb0d99d6124c9e53d36e/config/backup_disk.xml